Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -535,9 +557,13 @@ bool EventRegion::containsEditableElementsInRect(const IntRect& rect) const
 
 #if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
 
-void EventRegion::appendInteractionRegions(const Vector<InteractionRegion>& interactionRegions)
+void EventRegion::appendInteractionRegions(const Vector<InteractionRegion>& interactionRegions, const HashSet<ElementIdentifier>& containersToRemove)
```

Comment:
This feels like a weird shape for this function to take. We should either remove these regions before appending, or change the name to cullThenAppendInteractionRegions or something else like that. This has just taken on a very different shape, and I don't think I like it.

Reply:
Totally! Plus we should remove them before shrink wrapping anyway.